android/app/build.gradle

```diff
@@ -27,5 +27,6 @@ repositories {
 dependencies {
     implementation 'com.facebook.react:react-native:+'
     implementation "com.android.support:recyclerview-v7:${safeExtGet('supportLibVersion', '26.1.0')}"
-    implementation 'com.facebook.android:audience-network-sdk:5.+'
+    implementation 'com.facebook.android:audience-network-sdk:6.+'
+    implementation 'com.facebook.android:facebook-android-sdk:5.15.3'
```

Comment:
why do we use `6.+` in one dep and fixed `5.15.3` in another? I guess we want to be consistent and just in case have it fixed, at least to minor version

Reply:
I've pinned them to minor, but replaced the patch version with a `+`. So it's `6.2.+` and `5.15.+` now - What do you think?

Reply:
works for me